core/src/main/java/hudson/model/RunParameterDefinition.java

```diff
@@ -63,6 +63,7 @@ public String getName() {
     private final String runId;
     private final RunParameterFilter filter;
 
+    // TODO consider a simplified @DataBoundConstructor using @DataBoundSetter for description & filter
```

Comment:
A bit more complex in this case. Also less helpful since you would still have two mandatory attributes.